ci-operator/step-registry/openshift-qe/uperf-npt/OWNERS

```diff
@@ -0,0 +1,18 @@
+approvers:
```

Comment:
you'll want to delete this file and then create a symbolic link using the below to link to the OWNERS file in the openshift-qe base folder. That's where you should add your userid 
ln -s ./OWNERS ./<folder_name>

Reply:
I have fixed this